features/schema_metadata.feature

```diff
@@ -0,0 +1,303 @@
+      $session->execute(new Cassandra\SimpleStatement($cql));
+      $schema  = $session->schema();
+      $table   = $schema->keyspace("simplex")->table("values");
+
+      echo "Index Interval: " . $table->indexInterval() . "\n";
+      """
+    When it is executed
+    Then its output should contain:
+      """
+      Index Interval: 512
+      """
+
+    @cassandra-version-greater-1.2
+    Scenario: Additional table metadata will only exist for Cassandra versions
+              2.0+
+    Given the following example:
+      """php
+      <?php
+      $cluster = Cassandra::cluster()
+                         ->withContactPoints('127.0.0.1')
+                         ->build();
+      $session = $cluster->connect("simplex");
+      $cql = "ALTER TABLE values WITH default_time_to_live = '10000' AND "
+             . "memtable_flush_period_in_ms = '100' AND "
+             . "speculative_retry = '10ms'";
+      $session->execute(new Cassandra\SimpleStatement($cql));
+      $schema  = $session->schema();
+      $table   = $schema->keyspace("simplex")->table("values");
+
+      echo "Default TTL: " . $table->defaultTTL() . "\n";
+      echo "Memtable Flush Period: " . $table->memtableFlushPeriodMs() . "\n";
+      echo "Speculative Retry: " . intval($table->speculativeRetry()) . "\n";
+      """
+    When it is executed
+    Then its output should contain:
+      """
+      Default TTL: 10000
+      Memtable Flush Period: 100
+      Speculative Retry: 10
+      """
+
+    @cassandra-version-greater-2.0
+    Scenario: Additional table metadata will only exist for Cassandra versions
+              2.1+
+    Given the following example:
+      """php
+      <?php
+      $cluster = Cassandra::cluster()
+                         ->withContactPoints('127.0.0.1')
+                         ->build();
+      $session = $cluster->connect("simplex");
+      $cql = "ALTER TABLE values WITH max_index_interval = '16' AND "
+             . "min_index_interval = '4'";
+      $session->execute(new Cassandra\SimpleStatement($cql));
+      $schema  = $session->schema();
+      $table   = $schema->keyspace("simplex")->table("values");
+
+      echo "Maximum Index Interval: " . $table->maxIndexInterval() . "\n";
+      echo "Minimum Index Interval: " . $table->minIndexInterval() . "\n";
+      """
+    When it is executed
+    Then its output should contain:
+      """
+      Maximum Index Interval: 16
+      Minimum Index Interval: 4
+      """
+
+    @cassandra-version-greater-1.2
+    Scenario: Data types can be determined by the column metadata for Cassandra
+              versions 2.0+
+    Given the following example:
+      """php
+      <?php
+      $cluster   = Cassandra::cluster()
+                         ->withContactPoints('127.0.0.1')
+                         ->build();
+      $session   = $cluster->connect("simplex");
+      $schema    = $session->schema();
+      $id        = $schema->keyspace("simplex")->table("values")->column("id");
```

Comment:
I'd reduce this duplication by using something like `$values = $schema->keyspace("simplex")->table("values")`